### PR TITLE
add tests for wallet accounts

### PR DESCRIPTION
**Pull request by saledjenic**
Corresponding `status-go` PR:
- https://github.com/status-im/status-go/pull/3386

Tests are completed, they fully cover majority cases user may run into by adding/editing/removing wallet accounts.

The sad thing is that some of them randomly fail due to:
- right click menu for an already added wallet account fails or
- an option from that menu was missed

That's happening in `click_option_from_right_click_menu_of_account_with_name` function, where I've tried few different ways to overcome that, but each time even I've verify validity `if obj is None` (also `if obj == None`) and Squish successfully passed that check I was getting `Attempt to access a property of a null object` which is pretty weird. And I am not sure why it's like that.

An interesting thing is that they all passing well if I run them section by section in local, but if running them all some of them randomly fail, but always because of the issue described above.

Will mark tests here as `@mayfail` and create a new issue for Squish experts to investigate this.

### CODE REVIEW

File: test/ui-test/testSuites/suite_wallet/shared/steps/walletSteps.py [hunk 1]
```diff
@@ -15,36 +17,110 @@
...
+    _walletScreen.click_default_wallet_account()
 
+@When("the user adds a watch only account \"|any|\" with \"|any|\" color \"|any|\" and emoji \"|any|\" via \"|any|\"")
+@verify_screenshot
+def step(context, address, name, color, emoji, via_right_click_menu):
+    if via_right_click_menu == VALUE_YES:
+        _walletScreen.click_option_from_left_part_right_click_menu(MainWalletRightClickMenu.ADD_WATCH_ONLY_ACCOUNT_ACTION_PLACEHOLDER.value)
+    else:
+        _walletScreen.open_add_account_popup()
+    _walletScreen.add_account_popup_change_account_name(name)
```
Comment (noeliaSD) [inline test/ui-test/testSuites/suite_wallet/shared/steps/walletSteps.py:40]: Just minor suggestion. Up to you!
```suggestion
    _walletScreen.add_account_popup_change_account_settings(name, color,  emoji)
```

Comment (saledjenic) [reply test/ui-test/testSuites/suite_wallet/shared/steps/walletSteps.py:40]: I deliberately wanted to have separate function for each field which may be changed from the add/edit account popup, which gives us a possibility to write more granular tests.

File: test/ui-test/testSuites/suite_wallet/tst_wallet_section_accounts_management/test.feature [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+    As a user I want to add edit remove different types of wallet accounts
+
+    The feature start sequence is the following (setup on its own `bdd_hooks`):
+
+    ** given A first time user lands on the status desktop and generates new key
+    ** when user signs up with username "tester123" and password "TesTEr16843/!@00"
+    ** and the user lands on the signed in app
+    ** and the user opens the wallet section
+    ** and the user accepts the signing phrase
```
[the remark this answers is not in the document]
Comment (saledjenic) [reply test/ui-test/testSuites/suite_wallet/tst_wallet_section_accounts_management/test.feature:11]: All of those steps were already here, they are all already implemented and in use.

File: test/ui-test/src/screens/StatusWalletScreen.py [hunk 3]
```diff
@@ -98,81 +146,218 @@ class StatusWalletScreen:
...
+        
+    def remove_account_popup_do_cancel_action(self):
+        click_obj_by_name(RemoveAccountPopup.CANCEL_BUTTON.value)
+        
+    def remove_account_popup_do_remove_action(self, confirmHavingPenAndPaper, password: str):
+        if confirmHavingPenAndPaper:
+            click_obj_by_name(RemoveAccountPopup.HAVE_PEN_PAPER.value)
+        click_obj_by_name(RemoveAccountPopup.CONFIRM_BUTTON.value)
+        if password != NOT_APPLICABLE:
+            authenticatePopupEnterPassword(password)
```
[the remark this answers is not in the document]
Comment (saledjenic) [reply test/ui-test/src/screens/StatusWalletScreen.py]: @noeliaSD updated in the last commit.

**Review by alaibe — APPROVED** [review]

**Pull request merged**